### ISSUE TITLE
the unit test in ACSTest need to update

### ISSUE THREAD

**Issue by leaf-soba** [JabRef/jabref#11734]
### JabRef version

5.15 (latest release)

### Operating system

macOS

### Checked with the latest development build (copy version output from About dialog)

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. run unit test method `findByDOI` in `ACSTest`
- failed reason:the link of this case `https://pubs.acs.org/doi/pdf/10.1021/bk-2006-STYG.ch014` become "Page Not Found"
![2024-09-09 15 16 59](https://github.com/user-attachments/assets/34c1ca07-9f94-4113-aa4c-b1644328544b)
2. I checked all elements in [another page](https://pubs.acs.org/doi/abs/10.1021/acsphotonics.5b00001?journalCode=apchd5&quickLinkVolume=2&quickLinkPage=1&selectedTab=citation&volume=2) it don't have element `a.button_primary`, which is used in `org.jabref.logic.importer.fetcher.ACS` to find Fulltext PDF, maybe it need fix to use another method to check whether Fulltext PDF exists or not.

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

**Comment by Siedlerchr** [comment]
@koppor  You already took a look at the AcsFetcher, right?

**Comment by koppor** [comment]
Yes, used to try other scraping frameworks. See commits at https://github.com/koppor/jabref/pull/695
